services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -40,22 +39,48 @@
 logger = logging.getLogger(__name__)
 database_errors = (psycopg2.DatabaseError, asyncpg.exceptions.PostgresError)
 
+TASK_NAME = f"{__name__}.collect_garbage_periodically"
+TASK_CONFIG = f"{TASK_NAME}.config"
+
 
 def setup_garbage_collector(app: web.Application):
     async def _setup_background_task(app: web.Application):
-        # on_startup
+        # SETUP ------
         # create a background task to collect garbage periodically
-        loop = asyncio.get_event_loop()
-        cgp_task = loop.create_task(collect_garbage_periodically(app))
+        assert not any(  # nosec
+            t.get_name() == TASK_NAME for t in asyncio.all_tasks()
+        ), "Garbage collector task already running. ONLY ONE expected"  # nosec
```

Comment:
this is an assert to show "intention" (why spoke about that before, remember :-) )
this can happen if somebody unintentionally calls this setup function more than once
this actually happened in one of the tests where the tester wanted to create multiple clients (using ``aiohttp_client`` fixture) and inadvertently  created multiple ``app`` that scheduled multiple background garbage-collector tasks